### ISSUE TITLE
[BUG]: The techniques used by some groups seem to be missing

### ISSUE THREAD

**Issue by bontchev** [xakepnz/enterpriseattack#14]
### What happened?

Here is a small script to output the techniques used by a particular group:

```python
#!/usr/bin/env python


from __future__ import print_function


__description__ = 'Display the techniques used by an APT group'
__license__ = 'GPL'
__VERSION__ = '1.0.0'


from argparse import ArgumentParser
from enterpriseattack import Attack


def get_techniques(attack, group_id):
    for group in attack.groups:
        if group.id != group_id:
            continue
        techniques = []
        for technique in group.techniques:
            if technique.deprecated:
                continue
            if len(technique.sub_techniques):
                for subtechnique in technique.sub_techniques:
                    techniques.append(subtechnique.id)
            else:
                techniques.append(technique.id)
        return techniques


def main():
    parser = ArgumentParser(description=__description__)
    parser.add_argument('-v', '--version', action='version',
                        version='%(prog)s version {}'.format(__VERSION__))
    parser.add_argument('GID', nargs='+',
                        help='APT group ID')
    args = parser.parse_args()
    attack = Attack()
    for group_id in args.GID:
        techniques = get_techniques(attack, group_id)
        print('{}: {}'.format(group_id, ', '.join(techniques)))


if __name__ == '__main__':
    main()
```

If we use it for APT group [G0001](https://attack.mitre.org/groups/G0001/), it works fine:

    G0001: T1203, T1005, T1003.002, T1003.003, T1003.004, T1003.005, T1003.001, T1003.006, T1003.007, T1003.008, T1078.003, T1078.002, T1078.004, T1078.001, T1189, T1566.003, T1566.001, T1566.002, T1553.006, T1553.004, T1553.001, T1553.005, T1553.002, T1553.003, T1190, T1560.003, T1560.001, T1560.002

But if we use it for APT group [G0002](https://attack.mitre.org/groups/G0002/), the result is empty:

    G0002:

However, if we go to MITRE's site, we see that group [G0002](https://attack.mitre.org/groups/G0002/) is supposed to be using the technique `T1027.001`.

Maybe this is some deficiency in the data? Or is it the result of a data parsing bug?

### Version

0.1.6 (Default)

**Comment by xakepnz** [comment]
Hi, thanks again for finding this. I must've forgotten to add the `sub_techniques` property to the Group class.

Fixed in version 0.1.7 (#15)

Please try the following:

```bash
pip install enterpriseattack==0.1.7
```

Run the below to replicate your script:

```python
# --------------------------------------------------------------------------- #

import enterpriseattack
print(f'{enterpriseattack.__version__=}')

# --------------------------------------------------------------------------- #

# Update to ensure latest json, and do not include deprecated objects:
attack = enterpriseattack.Attack(
    include_deprecated=False,
    update=True
)

# --------------------------------------------------------------------------- #

# Print technique.id's and sub_technique.id's for the group:
def printTTPs(groupObj):
    # Merge the techniques and subtechniques into one list:
    mergedTechsAndSubs = groupObj.techniques + groupObj.sub_techniques
    print(
        f'{groupObj.id}: '
        f'{", ".join(t.id for t in mergedTechsAndSubs)}'
    )

# --------------------------------------------------------------------------- #

# Run against your examples:
yourExamples = ['G0001', 'G0002']

for group in attack.groups:
    if group.id in yourExamples:
        printTTPs(group)

# # Alternatively, test all the groups using this logic:
# for group in attack.groups:
#     printTTPs(group)
```

**Comment by bontchev** [comment]
Version `0.1.7` works correctly, thank you. Closing the issue now.
